### DISCUSSION TITLE
Add output for successful operation?

### DISCUSSION THREAD

**Discussion by Herve-M** [Tufin/oasdiff#368] [Ideas]
Hello Tufin Team,

After using oasdiff breaking check feature during CI, it looks hard to find out if the breaking / diff process was successful or not as no verbose feature is available. 

Example: 
- nothing breaking between A and A spec. 
  -  output: nothing 
- nothing breaking between A and B spec. 
  -  output: nothing 

Would be nice to have an output to state the operation was successful DX speaking. 

Thanks,

**Comment by reuvenharrison** [comment]
Hi,
You can use the `--fail-on` flag to get an exit status of 1 if there are breaking changes:
```
oasdiff breaking data/openapi-test1.yaml data/openapi-test3.yaml --fail-on ERR
oasdiff breaking data/openapi-test1.yaml data/openapi-test3.yaml --fail-on WARN
```
Then run `echo $?` to see the return code (0 or 1).

Regards,
Reuven

**Reply by Herve-M** [reply]
We can suppose that `oasdiff` has no silent error?

**Reply by Herve-M** [reply]
At first sight, it doesn't: 
`oasdiff breaking -c https://httpbun.org/status/404 https://raw.githubusercontent.com/Tufin/oasdiff/main/data/openapi-test1.yaml` or `oasdiff breaking -c https://raw.githubusercontent.com/Tufin/oasdiff/main/data/openapi-test1.yaml https://httpbun.org/status/404 `
return 0 
`oasdiff diff https://httpbun.org/status/404 https://raw.githubusercontent.com/Tufin/oasdiff/main/data/openapi-test1.yaml`
return `Error: failed to load base spec from "https://httpbun.org/status/404" with error loading "https://httpbun.org/status/404": request returned status code 404` and an errror code 102

**Comment by reuvenharrison** [comment]
Indeed, there is a bug.
This should return an error message:
```
oasdiff breaking -c https://httpbun.org/status/404 https://raw.githubusercontent.com/Tufin/oasdiff/main/data/openapi-test1.yaml
````

**Comment by reuvenharrison** [comment]
By the way, why are you using the `-c`?
This flag means: work in 'composed' mode, compare paths in all specs matching base and revision globs
But you are passing URLs, not globs.

**Comment by reuvenharrison** [comment]
See https://github.com/Tufin/oasdiff/issues/369
